_episodes/02-licensing.md

```diff
@@ -7,37 +7,67 @@ questions:
 - "When is a license important?"
 - "How to choose a license?"
 objectives:
-- "Add license to GitHub repository"
-- "Choose license with choosethelicense"
+- "Learn about software license"
+- "Add a license to a GitHub repository"
 keypoints:
-- "It is important to add a license to your open source code"
+- "It is important to add a license to your open-source code."
```

Comment:
Actually, it is not open source, when there is no license. Maybe 'It is important to add an open-source license to make your code open'.

Reply:
Done!